sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/azure/messaging/servicebus/perf/SendMessageTest.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.perf;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.servicebus.perf.core.ServiceBusStressOptions;
+import com.azure.messaging.servicebus.perf.core.ServiceTest;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Performance test.
+ */
+public class SendMessageTest extends ServiceTest<ServiceBusStressOptions> {
+    private final ClientLogger logger = new ClientLogger(SendMessageTest.class);
+    private Message message  = null;
```

Comment:
Either construct this in Semd Message Test COnstructor.
OR make this static so, multiple threads can refer to same message instance to be sent to SB.
If you make this static, then use it as a static reference in your test.